common/models/user.js

```diff
@@ -80,6 +80,9 @@ module.exports = function(User) {
       cb = options;
       options = undefined;
     }
+
+    cb = cb || app.createPromiseCallback();
```

Comment:
Where is the `app` variable defined? Perhaps this is throwing an error that is later converted to the error message you are seeing?